src/Campaigns/resources/admin/components/CampaignFormModal/index.tsx

```diff
@@ -127,9 +127,11 @@ export default function CampaignFormModal({isOpen, handleClose, apiSettings, tit
             goalType: campaign?.goalType ?? '',
             goal: campaign?.goal ?? null,
             startDateTime: getDateString(
-                campaign?.startDateTime?.date ? new Date(campaign.startDateTime.date) : getNextSharpHour(1)
+                campaign?.startDateTime?.date ? new Date(campaign?.startDateTime?.date) : getNextSharpHour(1)
+            ),
+            endDateTime: getDateString(
+                campaign?.endDateTime?.date ? new Date(campaign?.endDateTime?.date) : getNextSharpHour(2)
```

Comment:
@pauloiankoski we recently made the end date nullable so we should keep this default this to being empty (we don't want to generate a campaign end date)

Reply:
Reverted on 4c63c5bdbf44597fce7526adc701e206d6073d46